src/dataprotection/azext_dataprotection/manual/_params.py

```diff
@@ -0,0 +1,154 @@
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    get_enum_type,
+    resource_group_name_type,
+    get_location_type
+)
+from azure.cli.core.commands.validators import (
+    validate_file_or_dict
+)
+
+from azext_dataprotection.manual._validators import datetime_type, schedule_days_type
+from azext_dataprotection.manual.enums import (
+    get_protection_status_values,
+    get_backup_frequency_values,
+    get_datastore_type_values,
+    get_duration_type_values,
+    get_copy_option_values,
+    get_retention_rule_name_values,
+    get_absolute_criteria_values,
+    get_days_of_week_values,
+    get_weeks_of_month_values,
+    get_months_of_year_values,
+    get_tag_name_values,
+    get_job_status_values,
+    get_job_operation_values,
+    get_datasource_types
+)
+
+
+def load_arguments(self, _):
+    with self.argument_context('dataprotection backup-instance create') as c:
+        c.argument('backup_instance', type=validate_file_or_dict, help='Request body for operation Expected value: '
+                   'json-string/@json-file.')
```

Comment:
This argument should be flatten as it's not polymorphic class. Below are the arguments generated by CodeGen, if you want to modify this command manually, you can copy the generated code.

```python

    with self.argument_context('dataprotection backup-instance create') as c:
        c.argument('vault_name', type=str, help='The name of the backup vault.')
        c.argument('resource_group_name', resource_group_name_type)
        c.argument('backup_instance_name', options_list=['--name', '-n', '--backup-instance-name'], type=str,
                   help='The name of the backup instance')
        c.argument('friendly_name', type=str, help='Gets or sets the Backup Instance friendly name.')
        c.argument('data_source_info', action=AddDataSourceInfo, nargs='+', help='Gets or sets the data source '
                   'information.')
        c.argument('data_source_set_info', action=AddDataSourceSetInfo, nargs='+', help='Gets or sets the data source '
                   'set information.')
        c.argument('object_type', type=str, help='')
        c.argument('policy_id', type=str, help='', arg_group='Policy Info')
        c.argument('policy_parameters', action=AddPolicyParameters, nargs='+', help='Policy parameters for the backup '
                   'instance', arg_group='Policy Info')
```